Docsite-HTML-Files/slider.html

```diff
@@ -0,0 +1,237 @@
+          <li class="list-sidebar">
+            <h1 class="sidebar-heading">Get Started</h1>
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/install.html" class="routing"
+              >Installation
+            </a>
+          </li>
+          <li class="list-sidebar">
+            <h1 class="sidebar-heading">Components</h1>
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/Avatar.html" class="routing">
+              Avatar
+            </a>
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/Alert.html" class="routing"> Alert </a>
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/Button.html" class="routing">
+              Button
+            </a>
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/Card.html" class="routing"> Cards </a>
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/Badges.html" class="routing">
+              Badges
+            </a>
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/Modals.html" class="routing">
+              Drawer
+            </a>
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/Toast.html" class="routing"> Toast </a>
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/input.html" class="routing">
+              Inputs
+            </a>
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/Images.html" class="routing">
+              Images</a
+            >
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/Lists.html" class="routing"> Lists </a>
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/grids.html" class="routing">
+              Grids Simplified
+            </a>
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/typography.html" class="routing">
+              Typography
+            </a>
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/Modal.html" class="routing"> Modal </a>
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/Rating.html" class="routing">
+              Ratings
+            </a>
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/Navigation.html" class="routing">
+              Navigation
+            </a>
+          </li>
+          <li class="list-sidebar">
+            <a href="/Docsite-HTML-Files/slider.html" class="routing active">
+              Slider
+            </a>
+          </li>
+        </ul>
+      </aside>
+      <main class="Alert-wrapper">
```

Comment:
try to use consistent naming schemes, classnames usually start with a lower case letter